services/cocoapods/cocoapods.tester.js

```diff
@@ -4,14 +4,21 @@ const Joi = require('joi');
 const ServiceTester = require('../service-tester');
 
 const {
-    isMetric,
-    isMetricOverTimePeriod,
     isIntegerPercentage,
     isVPlusDottedVersionAtLeastOne,
 } = require('../test-validators');
 
 const isPlatform = Joi.string().regex(/^(osx|ios|tvos|watchos)( \| (osx|ios|tvos|watchos))*$/);
 
+// these are deliberately not isMetricOverTimePeriod due to
+// https://github.com/CocoaPods/cocoapods.org/issues/348
+const isMetricOverTimePeriodAllowZero = Joi
+  .string()
+  .regex(/^(0|[1-9][0-9]*)[kMGTPEZY]?\/(year|month|4 weeks|week|day)$/);
+const isMetricAllowZero = Joi
+  .string()
+  .regex(/^(0|[1-9][0-9]*)[kMGTPEZY]?$/);
```

Comment:
Would it be worth adding these to `test-validators`?

Reply:
Normally we only do this once more than one suite of tests use a validator, but I don't have particularly strong opinions on it..

Reply:
I think `npm`,  `hex.pm`, `packagist`, `source forge`, `DUB`, `package control` and `eclipse marketplace` all use the same style ![downloads-badge](https://img.shields.io/:downloads-1k/week-brightgreen.svg)
Although it looks like `npm` is missing test for `weekly/monthly/yearly` stats

Reply:
I think cocoapods is the only service where we want to allow this (unless we hit the same issue with one of those other services). We should continue to expect >0 in other cases as per the discussion in https://github.com/badges/shields/pull/1713#discussion_r192567128

Reply:
No worries, sounds good.
:+1: